R/filter-dm.R

```diff
@@ -1,19 +1,22 @@
 #' Filtering a [`dm`] object
 #'
-#' Filtering one table of a [`dm`] object may affect all tables connected to this table
-#' via one or more steps of foreign key relations. Firstly, one or more filter conditions for
-#' one or more tables can be defined using `cdm_filter()`, with a syntax similar to [`dplyr::filter()`].
-#' Each condition will only be immediately executed for the table it is referring to.
-#' Furthermore all conditions are stored in the [`dm`].
-#' With `cdm_apply_filters()` all tables will be updated according to the filter conditions and the foreign key relations.
+#' Filtering a table of a [`dm`] object may affect other tables that are connected to it
+#' directly or indirectly via foreign key relations.
+#'
+#' `cdm_filter()` can be used to define filter conditions for tables using syntax that is similar to `dplyr::filter()`.
```

Comment:
This will also add a link:

```suggestion
#' `cdm_filter()` can be used to define filter conditions for tables using syntax that is similar to [dplyr::filter()].
```